src/app.js

```diff
@@ -5,15 +5,10 @@ export default function (app) {
     return ""
   }
 
+  var emit = {event: event, hook: hook}
```

Comment:
This works, but it longer. You may help me get it shorter…

```js
function hook(type) {
  var args = Array.prototype.slice.call(arguments, 1);
  var result = args.pop();

  (subscriptions[type] || []).forEach(function(handler) {
    var val = handler.apply(null, args.concat(result))
    result = val !== undefined ? val : result
  })

  return result
}
```

Since hooks depend on the returned value and events don't, we need to check if the handler returns a value.